index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+        href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src="https://use.fontawesome.com/303b1d319c.js"></script>
+    <title>Portfolio</title>
+</head>
+
+<body>
+    <nav>
+        <div class="container0">
+            <a href="#" class='logo'>My Logo</a>
+            <a href="#" class="ham0">
+                <div class="ham"></div>
+                <div class="ham"></div>
+                <div class="ham"></div>
+            </a>
+        </div>
+    </nav>
+    <section>
+        <div class="container">
+            <p class="intro">I'm Hammas Glad to see you!</p>
+            <p class="description">I’m a software developer! I can help you build a product , feature or website Look
+                through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hestiate to contact me.</p>
+            <div class="container2">
+                <p class="connect">LET'S CONNECT</p>
+                <div class="icon-list">
```

Comment:
- The list of icons needs to use ``<ul>`` tag to group them, and ``<li>`` per icon; each icon has a link (``<a>`` tag).